### PR TITLE
Improves barbican barclamp description

### PR DESCRIPTION
**Pull request by sayalilunkad**

### CODE REVIEW

File: barbican.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@
 barclamp:
   name: 'barbican'
   display: 'Barbican'
-  description: 'OpenStack key and management service'
+  description: 'OpenStack Key Manager: Secure storage, provisioning and management of secret data'
```
Comment (vuntz) [inline barbican.yml]: I think I would do s/Manager/Management/ if it's still correct.

Comment (sayalilunkad) [inline barbican.yml]: I got the OpenStack key manager from http://docs.openstack.org/developer/barbican/ so maybe it's better to keep it the same?

**Comment by jgrassler** [comment]
+1 for "Key Manager". That's also the term we use in the service catalog.

File: barbican.yml (hunk 1 above)
Comment (vuntz) [inline barbican.yml]: Thanks, agreed.

**Comment by vuntz** [comment]
+1

**Pull request merged**